tests/lib/rules/accessor-pairs.js

```diff
@@ -1047,10 +1047,10 @@ ruleTester.run("accessor-pairs", rule, {
 
         // Full location tests
         {
-            code: "var o = { get a() {} };",
+            code: "var o = { get b() {} };",
```

Comment:
Is this test the same as some other one? If so, we can just remove it.

Reply:
This is what I meant trying to "resolve" the found duplicate test cases.
In this case there is a comment above that the following tests are for the error location.
As such I just renamed the identifier to keep the test cases intent.
The other options are just removing or merging the test cases.

Reply:
it sounds safe to remove the other one - it should have been covered by this one.

Reply:
All the duplicates were found because different error properties were tested, e.g. one test case for the message and one for the error location.
As such a part of the tests would be lost when deleting one of the test cases.
I thought the easiest solution would be to rename the identifier to keep the test case grouping intact.
Another solution would be to add the missing error properties for all the test cases.

Reply:
> Another solution would be to add the missing error properties for all the test cases.

👍 exactly what I mean to. :)

Reply:
adding `type` in this one, so that one can be removed.

https://github.com/eslint/eslint/blob/56d7198484d9f87b62fb1323174154a311fee305/tests/lib/rules/accessor-pairs.js#L636-L638

Reply:
> I thought the easiest solution would be to rename the identifier to keep the test case grouping intact.

It's more like a workaround, not a solution.

The question here is: do we allow/encourage users to write such tests?

Reply:
I think we should *allow* users to write these kinds of tests but not *encourage* it.
I think the majority of developers want to detect this but as I said there are various ways to resolve this and each developer may choose a different way.

Reply:
> The question here is: do we allow/encourage users to write such tests? If yes, we may not ignore the `errors`, instead of letting users use a workaround.

I think it's useful to detect tests that have the same input (code + options + config) because they might have been unintentional duplicates. And I think it's okay to use the workaround to modify the input a bit to avoid duplicates when the intent is to group test cases by differents kinds of assertions. We'll see feedback from users when we release this.